modules.d/99base/init.sh

```diff
@@ -16,20 +16,18 @@ PATH=/usr/sbin:/usr/bin:/sbin:/bin
 export PATH
 
 # mount some important things
-[ ! -d /proc/self ] \
-    && mount -t proc -o nosuid,noexec,nodev proc /proc > /dev/null
-
-if [ "$?" != "0" ]; then
-    echo "Cannot mount proc on /proc! Compile the kernel with CONFIG_PROC_FS!"
-    exit 1
+if [ ! -d /proc/self ]; then
+    if ! mount -t proc -o nosuid,noexec,nodev proc /proc > /dev/null; then
+        echo "Cannot mount proc on /proc! Compile the kernel with CONFIG_PROC_FS!"
```

Comment:
Probably a better wording here is 
`
echo "Cannot mount sysfs on /sys! Perhaps the kernel is not compiled with CONFIG_SYSFS?"
`
A subtle hint in a form of suggestion is better perceived than orders right.

Reply:
created an issue: https://github.com/dracutdevs/dracut/issues/1260